src/moveit_pro_kinova_configs/kinova_gen3_mujoco_config/objectives/demo_trajectory_stitching.xml

```diff
@@ -0,0 +1,144 @@
+        joint_names="{joint_names}"
+        velocities="{start_velocity}"
+      />
+      <Action
+        ID="CreateJointState"
+        positions="0.5;0.51;-3.025;-2.31;0.2;1.159;1.37"
+        joint_names="{joint_names}"
+        joint_state_msg="{target_state}"
+        velocities="0.0;0.0;0.0;0.0;0.0;0.0;0.0"
+      />
+      <Action
+        ID="GetRobotJointState"
+        planning_group_name="manipulator"
+        planning_scene="{planning_scene}"
+        joint_state="{joint_state}"
+      />
+      <Action
+        ID="CreateJointState"
+        positions="-0.5;-0.51;-3.025;-2.31;0.2;1.159;1.37"
+        joint_names="{joint_names}"
+        joint_state_msg="{stitch_target_state}"
+        velocities="0.0;0.0;0.0;0.0;0.0;0.0;0.0"
+      />
+      <Action
+        ID="GeneratePointToPointTrajectory"
+        jerk_scale_factor="0.5"
+        velocity_scale_factor="1.0"
+        acceleration_scale_factor="1.0"
+        planning_group_name="manipulator"
+        start_state="{start_state}"
+        start_time="0.000000"
+        target_state="{target_state}"
+        trajectory_sampling_rate="100"
+        joint_trajectory_msg="{joint_trajectory_msg}"
+      />
+      <Action
+        ID="GetTrajectoryStateAtTime"
+        time_from_reference="0.8"
+        joint_state="{stitch_state}"
+        from_start="true"
+        joint_trajectory_msg="{joint_trajectory_msg}"
+      />
+      <Action
+        ID="GeneratePointToPointTrajectory"
+        start_state="{stitch_state}"
+        target_state="{stitch_target_state}"
+        joint_trajectory_msg="{stitch_joint_trajectory_msg}"
+        velocity_scale_factor="1.0"
+        start_time="0.8"
+        acceleration_scale_factor="1.0"
+        jerk_scale_factor="0.5"
+        planning_group_name="manipulator"
+        trajectory_sampling_rate="100"
+      />
+      <Action
+        ID="SetAdmittanceParameters"
+        config_file_name="admittance_parameters.yaml"
+        admittance_parameters_msg="{admittance_parameters_msg}"
+      />
+      <Action
+        ID="ActivateControllers"
+        controller_names="/joint_trajectory_admittance_controller"
+      />
+      <Control ID="Parallel" failure_count="2" success_count="1">
+        <Action
+          ID="ExecuteTrajectoryWithAdmittance"
+          path_position_tolerance="0.2"
+          absolute_force_torque_threshold="45;45;45;10;10;10"
+          admittance_parameters_msg="{admittance_parameters_msg}"
+          controller_action_name="/joint_trajectory_admittance_controller/follow_joint_trajectory"
+          goal_duration_tolerance="-1.000000"
+          goal_position_tolerance="0.001000"
+          joint_trajectory_msg="{joint_trajectory_msg}"
+        />
+        <Decorator ID="Delay" delay_msec="400">
+          <Action
+            ID="ExecuteTrajectoryWithAdmittance"
+            joint_trajectory_msg="{stitch_joint_trajectory_msg}"
+            absolute_force_torque_threshold="450;450;450;100;100;100"
```

Comment:
I think this has to be so high because of a typo using 4ms instead of 8ms, and then the robot has to use a lot of jerk to align on the stitch trajectory. Does that sound likely or is there another reason to have different values?